src/admin/widgets/onboarding-flow/onboarding-flow.tsx

```diff
@@ -161,40 +159,53 @@ const OnboardingFlow = (props: WidgetProps) => {
 
   return (
     <>
-      <Container>
+      <Container className={clx(
+        "text-ui-fg-subtle px-0 pt-0 pb-4",
+        {
+          "mb-4": completed
+        }
+      )}>
         <Accordion
           type="single"
-          className="my-3"
           value={openStep}
           onValueChange={(value) => setOpenStep(value as STEP_ID)}
         >
-          <div className="flex items-center">
-            <div className="mr-5">
-              <GetStartedIcon />
-            </div>
+          <div className={clx(
+            "flex py-6 px-8",
+            {
+              "items-start": completed,
+              "items-center": !completed
+            }
+          )}>
+            <Avatar 
```

Comment:
nit: would replace this with a div with border-ui-border-base and use that to wrap the icon instead of using base64. This will also cause it to flicker from "M" to the image in some cases, as the fallback will show until the image is loaded.

Reply:
Good suggestion! I changed it to a div and an icon